truckersmp-cli

```diff
@@ -1,64 +1,81 @@
+
 
 class AppId:
     """ steam AppIds """
     game = {
-        "ats"         : 270880,        # https://steamdb.info/app/270880/
-        "ets2"        : 227300,        # https://steamdb.info/app/227300/
+        "ats":          270880,        # https://steamdb.info/app/270880/
+        "ets2":         227300,        # https://steamdb.info/app/227300/
     }
     proton = {
-        "4.2"         : 1054830,       # https://steamdb.info/app/1054830/
-        "3.16 Beta"   : 996510,        # https://steamdb.info/app/996510/
-        "3.16"        : 961940,        # https://steamdb.info/app/961940/
-        "3.7 Beta"    : 930400,        # https://steamdb.info/app/930400/
-        "3.7"         : 858280,        # https://steamdb.info/app/858280/
+        "4.2":          1054830,       # https://steamdb.info/app/1054830/
+        "3.16 Beta":    996510,        # https://steamdb.info/app/996510/
+        "3.16":         961940,        # https://steamdb.info/app/961940/
+        "3.7 Beta":     930400,        # https://steamdb.info/app/930400/
+        "3.7":          858280,        # https://steamdb.info/app/858280/
     }
     default_proton_ver = "4.2"
 
+
 class Dir:
     """ directories """
-    XDG_CACHE_HOME = os.getenv("XDG_CACHE_HOME",
-      os.path.join(os.path.expanduser("~"), ".cache"))
-    XDG_DATA_HOME = os.getenv("XDG_DATA_HOME",
-      os.path.join(os.path.expanduser("~"), ".local/share"))
+    XDG_CACHE_HOME = os.getenv(
+      "XDG_CACHE_HOME", os.path.join(os.path.expanduser("~"), ".cache"))
+    XDG_DATA_HOME = os.getenv(
+      "XDG_DATA_HOME", os.path.join(os.path.expanduser("~"), ".local/share"))
     default_gamedir = {
-        "ats" : os.path.join(XDG_DATA_HOME,
-          "truckersmp-cli/American Truck Simulator/data"),
-        "ets2" : os.path.join(XDG_DATA_HOME,
-          "truckersmp-cli/Euro Truck Simulator 2/data"),
+        "ats": os.path.join(
+          XDG_DATA_HOME, "truckersmp-cli/American Truck Simulator/data"),
+        "ets2": os.path.join(
+          XDG_DATA_HOME, "truckersmp-cli/Euro Truck Simulator 2/data"),
     }
     default_prefixdir = {
-        "ats" : os.path.join(XDG_DATA_HOME,
-          "truckersmp-cli/American Truck Simulator/prefix"),
-        "ets2" : os.path.join(XDG_DATA_HOME,
-          "truckersmp-cli/Euro Truck Simulator 2/prefix"),
+        "ats": os.path.join(
+          XDG_DATA_HOME, "truckersmp-cli/American Truck Simulator/prefix"),
+        "ets2": os.path.join(
+          XDG_DATA_HOME, "truckersmp-cli/Euro Truck Simulator 2/prefix"),
     }
     default_moddir = os.path.join(XDG_DATA_HOME, "truckersmp-cli/TruckersMP")
     default_protondir = os.path.join(XDG_DATA_HOME, "truckersmp-cli/Proton")
     scriptdir = os.path.dirname(os.path.realpath(__file__))
     steamcmddir = os.path.join(XDG_CACHE_HOME, "truckersmp-cli/steamcmd")
 
 
-def printdbg(msg, end = "\n"):
+def printdbg(msg, end="\n"):
     if args.verbose:
-        print(msg, end = end)
+        print(msg, end=end)
```

Comment:
printdbg should also send the messages to stderr.

stdout should only be used for the actual output of the program, the kind of output that would be then parsed by a subsequent pipe.